src/app/components/layout/header/header.component.ts

```diff
@@ -91,11 +92,13 @@ export class HeaderComponent implements OnInit, OnDestroy {
     this.coinSubscription.unsubscribe();
   }
 
-  private resetBalance() {
+  private reloadBalance() {
     this.coins = 0;
     this.price = null;
     this.balance = null;
     this.isBalanceLoaded = false;
+
```

Comment:
Please, add this to avoid some errors in the UI:

```
this.isBlockchainLoading = true;
this.percentage = null;
this.current = null;
this.highest = null;
```